packages/block-library/src/media-text/media-container.native.js

```diff
@@ -168,116 +169,135 @@ class MediaContainer extends Component {
 
 	renderImage( params, openMediaOptions ) {
 		const { isUploadInProgress } = this.state;
-		const { mediaAlt, mediaUrl, isSelected } = this.props;
 		const {
-			finalWidth,
-			finalHeight,
-			imageWidthWithinContainer,
-			isUploadFailed,
-			retryMessage,
-		} = params;
-		const opacity = isUploadInProgress ? 0.3 : 1;
-
-		const contentStyle = ! imageWidthWithinContainer
-			? styles.content
-			: styles.contentCentered;
+			aligmentStyles,
+			focalPoint,
+			imageFill,
+			isMediaSelected,
+			isSelected,
+			mediaAlt,
+			mediaUrl,
+			mediaWidth,
+			shouldStack,
+		} = this.props;
+		const { isUploadFailed, retryMessage } = params;
+		const focalPointValues =
+			imageFill && ! focalPoint ? { x: 0.5, y: 0.5 } : focalPoint;
```

Comment:
WDYT about creating const like `DEFAULT_FOCAL_POINT`?

Reply:
Cool! I moved it to a const.